test/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests/OpenTelemetry.Contrib.Extensions.AWSXRay.Tests.csproj

```diff
@@ -3,12 +3,16 @@
   <PropertyGroup>
     <Description>Unit test project for AWS X-Ray for OpenTelemetry</Description>
     <TargetFrameworks>netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net452</TargetFrameworks>
```

Comment:
shouldn't you test net452 as well?

Reply:
Yes. I'll add the tests for net452 soon. Its just that creating a mock http layer for net45 as well is cumbersome. :(
Also looking at some alternatives to doing this.

Reply:
I tried using [WireMock.Net]() to stub the underlying http requests made by AWS SDK client, but couldn't get it to work. 
For now I've added the mock http layer code for net452 alongwith the previous code for netcore.